Wrappers/Python/cil/framework/framework.py

```diff
@@ -1927,9 +1927,14 @@ def angle_unit(self,val):
     def __str__(self):
         repres = "Acquisition description:\n"
         repres += "\tNumber of positions: {0}\n".format(self.num_positions)
-        num_print=min(20,self.num_positions)    
-        repres += "\tAngles 0-{0} in {1}s:\n{2}\n".format(num_print, self.angle_unit, numpy.array2string(self.angle_data[0:num_print], separator=', '))
-        return repres   
+        # max_num_print = 30 
+        if self.num_positions < 31:
+            repres += "\tAngles 0-{0} in {1}s: {2}\n".format(self.num_positions, self.angle_unit, numpy.array2string(self.angle_data[0:self.num_positions], separator=', '))
```

Comment:
```suggestion
            repres += "\tAngles 0-{0} in {1}s: {2}\n".format(self.num_positions-1, self.angle_unit, numpy.array2string(self.angle_data[0:self.num_positions], separator=', '))
```

Reply:
Sorry, another change but say we have num_positions = 30, it should say angles 0-29

Reply:
Also I don't think there was a unit test for this case